pkg/module_manager/models/modules/basic.go

```diff
@@ -627,7 +627,7 @@ func (bm *BasicModule) executeHook(h *hooks.ModuleHook, bindingType sh_op_types.
 		bm.dc.MetricStorage.HistogramObserve("{PREFIX}module_hook_run_user_cpu_seconds", hookResult.Usage.User.Seconds(), metricLabels, nil)
 		bm.dc.MetricStorage.GaugeSet("{PREFIX}module_hook_run_max_rss_bytes", float64(hookResult.Usage.MaxRss)*1024, metricLabels)
 	}
-	if err != nil {
+	if err != nil || hookResult == nil {
```

Comment:
One line below we have the check
```
if hookResult != nil && ...
```
It's a bit weird, to pass the check `hookResult == nil` and next line check `hookResult != nil`